conduit/src/Data/Conduit/Internal/Conduit.hs

```diff
@@ -721,8 +724,11 @@ fuse = (=$=)
 --
 -- Output from the upstream (left) conduit will be fed into the
 -- downstream (right) conduit. Processing will terminate when
--- downstream (right) returns. Leftover data returned from the right
--- @Conduit@ will be discarded.
+-- downstream (right) returns, and both @Conduit@s will be closed.
```

Comment:
The "closed" comment is no longer relevant in 1.3 due to the removal of finalizers.